schema/Retail/RET/on_init.ts

```diff
@@ -86,22 +86,8 @@ export const onInitSchema = {
                   type: 'string',
                   minLength: 1,
                 },
-                locations: {
```

Comment:
This is mandatory now. provider_location is deprecated and locations property is moved here in /provider